meeting.css

```diff
@@ -0,0 +1,167 @@
+/* General Reset */
+body, html {
+    width: 100%;
+    height: 100%;
+    font-family: 'Times New Roman', Times, serif;
+    background-color: black;
+    margin: 0;
+    padding: 0;
+    color: white; /* Set text color to white */
+}
+
+#body{
+    background-color: black;
+}
+
+/* Container for the Join button */
+.container3 {
+    position: absolute;
+    background-color: black;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%); /* Center the container */
+}
+
+/* Style for the Join button */
+#joinMeetingButton {
+    color: white;
+    font-weight: 600;
+    text-align: center;
+    cursor: pointer;
+    border-radius: 5px;
+    border: none;
+    padding: 10px 20px;
+    background-color: rgb(46, 160, 8);
+    font-size: 1.5rem;
+    transition: background-color 0.3s ease; /* Smooth background color change on hover */
+}
+
+#joinMeetingButton:hover {
+    background-color: rgb(46, 160, 8); /* Darken button color on hover */
+}
+
+/* Modal Content Styling */
+.modal-content {
+    background-color: black;
+    border-radius: 25px;
+    border: none;
+    width: 150%; /* Set width to 1000px */
+    height: 570px; /* Set height to 570px */
+    margin: auto; /* Center the modal box */
+    position: relative; /* Position relative for child positioning */
+    top:50%;
+}
+#modal1{
+    width: 1000px;
+    height: 617px;
+    position: relative;
+    top:50px;
+    left: -12%;
+    border-radius: 20px;
+    background-color: rgb(96, 96, 96);
+}
+/* Remote Video Styles */
+#remoteVideo {
+    width: 912px; /* Set width */
+    height: 513px; /* Set height */
+    border-radius: 25px;
+    margin: 5px auto 0;
+    position: fixed;
+    left: 24%; /* Center video and position it 5px from the top */
+}
+
+/* Modal Body Styling */
+.modal-body {
+    position: relative; /* Set position to relative for absolute positioning of local video */
+}
+
+/* Leave and Unmute Buttons */
+.d-flex {
+    display: flex;
+    justify-content: center; /* Center the buttons */
+}
+
+/* Unmute Button Style */
+#Microphone {

```

Comment:
Please ensure to keep all the variable names starting with a lower case alphabet